src/parachain/issue.ts

```diff
@@ -218,24 +218,15 @@ export class DefaultIssueAPI extends DefaultTransactionAPI implements IssueAPI
         amountsPerVault: Map<AccountId, Big>,
         atomic: boolean
     ): Promise<IssueRequestResult[]> {
-        const txes = new Array<SubmittableExtrinsic<"promise">>();
-        console.log("USING VAULTS on ISSUE:");
-        [...amountsPerVault.entries()].map(([vaultId, amount]) => {
-            console.log(`Issuing ${amount.toString()} with vault ${vaultId.toString()}`);
-        });
+        const txs = new Array<SubmittableExtrinsic<"promise">>();
         for (const [vault, amount] of amountsPerVault) {
-            const griefingCollateral = await this.getGriefingCollateral(amount);
-            const amountWrapped = this.api.createType("Wrapped", btcToSat(amount.toString()));
-            console.log(`Issuing:
-                - amountSat: ${amountWrapped.toString()}
-                - to vaultId: ${vault.toString()}
-                - with griefing collateral: ${griefingCollateral.toString()}`);
-            txes.push(this.api.tx.issue.requestIssue(amountWrapped, vault, dotToPlanck(griefingCollateral.toString()) as string));
+            const griefingCollateral = (await this.getGriefingCollateral(amount)).add(1);
```

Comment:
Quick question, why `add(1)`? Especially since this is still the DOT value, this looks like it will inflate all griefing collaterals by 1 DOT.

Reply:
Yeah this was a quick hack, there is some discrepancy between this calculation of the griefing collateral and that on-chain. Likely related to the lose of precision going to and from Planck - DOT.

Reply:
Have you tested if your rounding up changes fix that - or should we add a few Planck instead?